src/controllers/event/add.ts

```diff
@@ -118,15 +148,41 @@ const createEvent: Interfaces.Controller.Async = async (req, res, next) => {
       module: {
         connect: { id: moduleId },
       },
-      organizers: {
-        connect: organizersUsernames,
-      },
-      managers: {
-        connect: managersUsernames,
-      },
     },
   });
 
+  if (organizers) {
+    try {
+      organizers?.map(async (id: any) => {
+        await prisma.eventOrganiser.create({
+          data: {
+            userId: id,
+            eventId: event.id,
+          },
```

Comment:
Just to make sure, is this the correct way to create and relate entities in prisma ?

can you confirm ?

Reply:
Yes bhaiya it is working